spec/controllers/admin/organizations_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Admin::OrganizationsController do
+  let(:admin) { create(:admin) }
```

Comment:
I think we're missing some tests to make sure that only admins have access to these routes.